src/appforms/tests/modules.js

```diff
@@ -43,7 +44,35 @@
   "060-RuleEngine"
  ],
  backbone:[
-  "040-view06Steps"
```

Comment:
I'm sure I'm missing something here but some of the test listed here do not exist as far as I can tell?